src/Core/Framework/Plugin/PluginLifecycleService.php

```diff
@@ -141,11 +141,13 @@ public function installPlugin(PluginEntity $plugin, Context $shopwareContext): I
             return $installContext;
         }
 
-        // TODO NEXT-1797: Not usable with Composer 1.8, Wait for Release of Composer 2.0
-        if (next1797()) {
-            $this->executor->require($plugin->getComposerName());
-        } else {
-            $this->requirementValidator->validateRequirements($pluginBaseClass, $shopwareContext, 'install');
+        if ($plugin->isManagedByComposer()) {
```

Comment:
This check should actually only be done in the `else` case, since the code in the `next1797()` case will actually require the plugin (causing the plugin to become "managed by composer").